<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditTownDialog</class>
 <widget class="QDialog" name="EditTownDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>town</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="townLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_country">
         <property name="text">
          <string>country</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="countryLineEdit"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="applyPushButton">
           <property name="text">
            <string>apply</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deletePushButton">
           <property name="text">
            <string>delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="closePushButton">
           <property name="text">
            <string>close</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
